cmd/relay-gossip/main.go

```diff
@@ -150,70 +113,12 @@ var runCmd = &cli.Command{
 }
 
 var clientCmd = &cli.Command{
```

Comment:
does this do anything different from the `client` binary at this point? can we remove /deprecate it fully if it is purely redundant?

Reply:
Should we just make the client binary also be a gossip relay via option switch. Because a gossip-relay is ideally a full client + gossiping function turned on.

Reply:
> does this do anything different from the `client` binary at this point? can we remove /deprecate it fully if it is purely redundant?

No, it just allows you to look at randomness without having to compile and use a separate binary.